frontend/src/app/components/filters/quick-filter-by-text-input/quick-filter-by-text-input.component.ts

```diff
@@ -29,13 +29,13 @@
 import {Component, EventEmitter, Output} from '@angular/core';
 import {I18nService} from "app/modules/common/i18n/i18n.service";
 import {WorkPackageViewFiltersService} from "core-app/modules/work_packages/routing/wp-view-base/view-services/wp-view-filters.service";
-import {WorkPackageCacheService} from "app/components/work-packages/work-package-cache.service";
 import {Subject} from "rxjs";
 import {debounceTime, distinctUntilChanged, map, tap} from "rxjs/operators";
 import {IsolatedQuerySpace} from "core-app/modules/work_packages/query-space/isolated-query-space";
 import {input} from "reactivestates";
 import {QueryFilterResource} from "core-app/modules/hal/resources/query-filter-resource";
 import {UntilDestroyedMixin} from "core-app/helpers/angular/until-destroyed.mixin";
+import {APIV3Service} from "core-app/modules/apiv3/api-v3.service";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/e71869ec897476f98b23dceb4c3c60722bb3632a/frontend/src/app/components/filters/quick-filter-by-text-input/quick-filter-by-text-input.component.ts#L39"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>